tests/DuplexResourceStreamTest.php

```diff
@@ -56,7 +56,7 @@ public function testConstructorThrowsExceptionOnInvalidStream()
     {
         $loop = $this->createLoopMock();
 
-        $this->setExpectedException('InvalidArgumentException');
+        $this->expectException('InvalidArgumentException');
```

Comment:
Definitely reasonable! I would suggest updating this to the class syntax while we're at it (see also other changes): 

```suggestion
        $this->expectException(\InvalidArgumentException::class);
```

Reply:
Hah doh, will also have a look at this for https://github.com/reactphp/event-loop/pull/276 /cc @SimonFrings